traffic_ops/traffic_ops_golang/federation_resolvers/federation_resolvers.go

```diff
@@ -248,6 +249,24 @@ func Delete(w http.ResponseWriter, r *http.Request) {
 	}
 	defer inf.Close()
 
+	fedID, err := strconv.Atoi(inf.Params["id"])
```

Comment:
I think we can do `fedID := inf.IntParams["id"]` here since `id` is required and an int param (per L244). Then we don't have a conversion error to handle.